runtime/lua/vim/filetype.lua

```diff
@@ -851,7 +851,7 @@ local extension = {
   stm = function() vim.fn["dist#ft#FThtml"]() end,
   tcsh = function() vim.fn["dist#ft#SetFileTypeShell"]("tcsh") end,
   tex = function() vim.fn["dist#ft#FTtex"]() end,
-  tf = function() vim.fn["dist#ft#FTtf"]() end,
+  tf = function() return require'vim.filetype.detect'.tf() end,
```

Comment:
Style nitpick. I don't really care about single or double quotes, as long as we're consistent throughout the file.

I think we should also establish a precedent of always passing `path` and `bufnr` to the detection functions, even if they're not used.

```suggestion
  tf = function(path, bufnr) return require("vim.filetype.detect").tf(path, bufnr) end,
```

Reply:
(We should run those new files through Stylua, anyway.)